templates/profiler/seo_collector.html.twig

```diff
@@ -115,15 +115,43 @@
                 {# Todo : est-ce possible de récupérer le texte brut ici ⬇ ? #}
                 <td class="font-normal">
                     <div>{{ profiler_dump(collector.seek('title')) }}</div>
-                    {# <div class="text-muted">30 characters (recommended : less than 60 characters)</div> #}
+                    {% if collector.title|length == 0 %}
+                        <div class="card status-error">
+                            <strong>Warning : missing title (recommended : 50 - 60 characters)</strong>
+                        </div>
+                    {% elseif collector.title|length < 50 %}
```

Comment:
C'est vraiment 50 à 60 chars ? ça laisse peu de marge, la limite basse est étonnante. Pas sûr qu'il faille afficher ça comme une "erreur" si le titre fait moins de 50 chars, si ? Eventuellement mettre en évidence si c'est vraiment trop court (20 chars ?).

(j'ai aucune idée des recos exactes hein)

Reply:
je me réfère aux recommandations de moz.com qui fait un peu autorité dans le domaine, voilà ce qu'ils disent :

> Google typically displays the first 50–60 characters of a title tag. If you keep your titles under 60 characters, our research suggests that you can expect about 90% of your titles to display properly. There's no exact character limit, because characters can vary in width and Google's display titles max out (currently) at 600 pixels.

https://moz.com/learn/seo/title-tag

Donc si ton title est composé de 'm', bah t'en mets moins que des 'i', mais bon 🤓

Reply:
D'ailleurs c'est pareil, on affichera un warning si c'est 61 caractères, c'est un peu ridicule. Au pire j'enlève tous les warning (sauf si pas de title)

Reply:
Ma remarque portait surtout sur la limite basse étant si proche de la limite haute, ce qui était suspicieux. De ce que je comprend de mon côté, c'est surtout qu'il y a une "limite haute" conseillée de 50 à 60 chars, car l'affichage par Google diffère selon ls chars, mais sera tjrs au max 60 chars:

> Google typically displays the first 50–60 characters of a title tag. If you keep your titles under 60 characters, our research suggests that you can expect about 90% of your titles to display properly. There's no exact character limit, because characters can vary in width and Google's display titles max out (currently) at 600 pixels.

Donc en fait, y a pas de limite basse (mais certainement des reco quelque part, plus de l'ordre de 20-30 chars j'imagine).

> D'ailleurs c'est pareil, on affichera un warning si c'est 61 caractères, c'est un peu ridicule. Au pire j'enlève tous les warning (sauf si pas de title)

Du coup non, la limite haute semble justifiée. C'est juste la tournure 50 - 60 qui n'était pas à comprendre comme un range, mais une estimation 😄